connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectWorkerIntegrationTest.java

```diff
@@ -1375,6 +1380,57 @@ public void testRuntimePropertyReconfiguration() throws Exception {
         );
     }
 
+    @Test
+    public void testPluginAliases() throws Exception {
+        connect = connectBuilder.build();
+        // start the clusters
+        connect.start();
+
+        // Create a topic; not strictly necessary but prevents log spam when we start a source connector later
+        final String topic = "kafka17510";
```

Comment:
I guess the correct jira number is "17150"

Reply:
Your eye for detail never ceases to impress me 😄
Fixed in the latest commit.